.gitignore

```diff
@@ -21,4 +21,5 @@ MicroEngineerProject/[Bb]in/
 /MicroEngineerProject/.vs
 /MicroEngineerProject/.vs/*
 /Staging/CopyStagingToGame.bat
-*.bat
\ No newline at end of file
+*.bat
+*.csproj
```

Comment:
What's the reason for this addition? Project files should not be ignored.

Reply:
Just temporary so my github desktop ignores the project files I've edited for my setup. Will remove this once PR is ready.